docs/intro/tutorial.rst

```diff
@@ -83,44 +73,70 @@ outside our spider file.
             }
 
 Now we have a ``BookPage`` class that implements the ``to_item`` method.
-We could go even further and extract a property from the ``to_item`` method.
+This class contains all logic necessary for extracting an item from
+an individual book page like
+http://books.toscrape.com/catalogue/a-light-in-the-attic_1000/index.html,
+and nothing else.
+
+If we want, we can organize code in a different way, and e.g.
+extract a property from the ``to_item`` method:
 
 .. code-block:: python
 
     from web_poet.pages import ItemWebPage
 
 
     class BookPage(ItemWebPage):
-        """Extracts data from a book page"""
+        """Individual book page on books.toscrape.com website"""
 
         @property
         def title(self):
-            """Extracts title from book page"""
+            """Book page title"""
             return self.css("title::text").get()
 
         def to_item(self):
-            """Converts page into an item"""
             return {
                 'url': self.url,
                 'name': self.title,
             }
 
+Configuring the project
+=======================
+
+To use scrapy-poet, enable its downloader middleware in ``settings.py``:
+
+.. code-block:: python
+
+    DOWNLOADER_MIDDLEWARES = {
+        'scrapy_poet.InjectionMiddleware': 543,
+    }
+
+
+BookPage class we created previously can be used without scrapy-poet,
+and even without Scrapy (note that imports were from ``web_poet`` so far).
+
+``scrapy-poet`` makes it easy to use ``web-poet`` Page Objects
+(such as BookPage) in Scrapy spiders.
+
 Changing spider
 ===============
 
-The next step is to change our ``parse_book`` method in order to consume our
-newly created Page Object class.
+To use the newly created BookPage class in the spider, change
+the ``parse_book`` method as follows:
 
 .. code-block:: python
 
-    def parse_book(self, response, book_page: BookPage):
-        """Extract data from book pages"""
-        yield book_page.to_item()
+    class BooksSpider(scrapy.Spider):
+        # ...
+        def parse_book(self, response, book_page: BookPage):
+            """Extract data from book pages"""
+            yield book_page.to_item()
 
-The parser method now receives a type annotated argument called ``book_page``.
-Our Injection Middleware will detect it and provide the required dependencies.
+``parse_book`` method now has a type annotated argument
+called ``book_page``. scrapy-poet detects this and makes sure
+a BookPage instance is created and passed to the callback.
 
-The spider should be looking like this:
+The full spider code would be looking like this:
 
 .. code-block:: python
 
```

Comment:
Are we importing scrapy here as well?

Reply:
added an import, thanks! Initially I haven't added it because this is not a complete example (it needs a BookPage to work).